### DISCUSSION TITLE
Queryparams support in the cms?

### DISCUSSION THREAD

**Discussion by mintyPT** [wagtail/wagtail#12597] [Q&A]
I looked online, tried a couple of things but got no where.

Is there a way to pass information via the url to prefill some of the fields in the cms edit interface?

For example, opening this url

`https://thebeatshelter.com/cms/pages/add/.../7/?title=ABC`

Would fill the title field with ABC

![CleanShot 2024-11-19 at 08 26 47](https://github.com/user-attachments/assets/01d6fffa-4a21-4f75-b2b0-3430cddbfbc7)

**Comment by gasman** [comment]
I don't think there's a good way of doing this yet. This was one of the motivations for https://github.com/wagtail/wagtail/issues/12463, but work on that has stalled.

**Reply by mintyPT** [reply]
That's a shame. It would be highly beneficial.
